File: Strings/slides/representacao_de_strings/codes/io.py
# -*- coding: utf-8 -*-

# Assuma que será inserida em uma linha, via console, a mensagem
# "Teste de I/O em Python"

s = input()
print(f's = [{s}]')         # s = [Teste de I/O em Python]

s = s.split()[0]
print(f's = {s}')           # s = Teste

File: Strings/slides/representacao_de_strings/codes/io.cpp
#include <iostream>

using namespace std;

// Assuma que será inserida em uma linha, via console, a mensagem
// "Teste de I/O em C++"
int main()
{
    string s, line;

    cin >> s;
    cout << "s = [" << s << "]\n";              // s = [Teste]

    getline(cin, line);
    cout << "line = [" << line << "]\n";        // line = [ de I/O em C++]

    return 0;
}
